scipy/signal/signaltools.py

```diff
@@ -361,34 +366,59 @@ def fftconvolve(in1, in2, mode="full"):
     in1 = asarray(in1)
     in2 = asarray(in2)
 
+    if axes is None:
+        pass  # skip next few checks
+    elif not axes:
+        raise ValueError('if axes is specified it must be non-empty')
```

Comment:
There can be other bad arguments, such as `axes=False` where this message becomes confusing.